README.md

```diff
@@ -28,7 +28,7 @@ in WeeChat.
 
 #Support
 
-IRC channel: #weechat-otr on Freenode.
+IRC channel: [#weechat-otr on Freenode](ircs://chat.freenode.net:+6697).
```

Comment:
Shouldn't it be ircs://chat.freenode.net:6697 (without the plus)?
https://www.iana.org/assignments/uri-schemes/prov/ircs